core/src/test/java/com/yahoo/imapnio/async/client/ImapFutureTest.java

```diff
@@ -60,7 +60,7 @@ public void beforeMethod() throws IOException, ProtocolException, ImapAsyncClien
         final ImapRequest imapRequest = new AuthXoauth2Command("username", "token", new Capability(capas));
         final int requestTotalBytes = "a1".getBytes(StandardCharsets.US_ASCII).length + 1 + imapRequest.getCommandLineBytes().readableBytes();
         imapAsyncResp = new ImapAsyncResponse(imapRequest.getCommandType(), requestTotalBytes,
-                oneImapResponse.toString().getBytes(StandardCharsets.US_ASCII).length + 2, imapResponses);
+                oneImapResponse.toString().getBytes(StandardCharsets.US_ASCII).length + 2, imapResponses, null);
```

Comment:
Create the total elapsed time for the IMAPResponse and perform an assertion to validate the elapsed time.